src/lib/drivers/gyroscope/PX4Gyroscope.cpp

```diff
@@ -121,6 +123,15 @@ void PX4Gyroscope::update(hrt_abstime timestamp, int16_t x, int16_t y, int16_t z
 	// Filtered values
 	const matrix::Vector3f val_filtered{_filter.apply(val_calibrated)};
 
+	// publish control data (filtered gyro) immediately
+	sensor_gyro_control_s &control = _sensor_gyro_control_pub.get();
+	control.timestamp_sample = timestamp;
+	control.x = val_filtered(0);
```

Comment:
An array in the message would allow for `val_filtered.copyTo(control.angular_velocity);`

Reply:
Done.